.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -933,4 +933,10 @@ else
         python3 -u ${SUBMITTY_INSTALL_DIR}/sbin/restart_shipper_and_all_workers.py
         echo -e -n "Done restarting shipper & workers\n\n"
     fi
+
+    # Update OS / Docker versions for all machines
+    echo "Update OS and Docker versions for all machines"
+    DATE_YMD=`date +%Y%m%d`
+    echo -e '{\n  "job": "UpdateDockerImages"\n}' > ${SUBMITTY_DATA_DIR}/daemon_job_queue/docker${DATE_YMD}.json
```

Comment:
It is good practice to quote variables in shell scripts.  It's probably not too big of a risk here, but it can cause a bunch of [unexpected problems](https://unix.stackexchange.com/a/171347/195445) if there are spaces or other special characters in the variable.
```suggestion
    echo -e '{\n  "job": "UpdateDockerImages"\n}' > "${SUBMITTY_DATA_DIR}/daemon_job_queue/docker${DATE_YMD}.json"
```